crates/radix-engine-toolkit-uniffi/src/derive/functions.rs

```diff
@@ -80,7 +100,7 @@ pub fn derive_package_of_direct_caller_non_fungible_global_id_from_component_add
     network_id: u8,
 ) -> Result<Arc<NonFungibleGlobalId>> {
     let package_address = NativePackageAddress::try_from(*package_address)?;
-    let non_fungible_global_id = core_package_of_direct_caller_non_fungible_global_id_from_component_address(package_address);
+    let non_fungible_global_id = core_package_of_direct_caller_non_fungible_global_id_from_package_address(package_address);
```

Comment:
The method name here should also be changed I guess? `derive_package_of_direct_caller_non_fungible_global_id_from_component_address` => `derive_package_of_direct_caller_non_fungible_global_id_from_package_address`